__tests__/html-document.js

```diff
@@ -73,3 +73,23 @@ test('.document() - js "type" is "esm" - should set type to module on script tag
     const result = document(incoming);
     expect(result).toMatchSnapshot();
 });
+
+test('.document() - js "type" is "fallback" - should wrap fallback scripts in ie comment block', () => {
+    const incoming = new HttpIncoming(SIMPLE_REQ, SIMPLE_RES);
+    incoming.css = [
+        { value: 'http://somecssurl1.com', type: 'text/css' },
+        { value: 'http://somecssurl2.com', type: 'text/css' },
+        { value: 'http://somecssurl3.com', type: 'text/css' },
+    ];
+    incoming.js = [
+        { value: 'http://somejsurl1.com/main', type: 'module' },
+        { value: 'http://somejsurl1.com/ie11', type: 'fallback' },
+        { value: 'http://somejsurl2.com/main', type: 'module' },
+        { value: 'http://somejsurl2.com/ie11', type: 'fallback' },
```

Comment:
`type` is currently using the same value as the `type` attribute on the `<script>` and `<link>` tags in HTML. `fallback` will diverge from that. How do one define for a fallback that the script is of type `module`?

Reply:
Yea, tis a good point. I don't think this is a problem for Finn as we can just call it whatever we like in our custom document template, however its a bit less than ideal if we introduce the new asset pipe and it doesn't work optimally with Podium so it feels to me like we should have a good story in place for this case. It could be that we do it at the Podium end as this approach proposes or it could be that APv3 doesn't produce fallback bundles by default (maybe thats custom). On that note, the esm + ie11 bundle approach strikes me as a little too rigid. I would guess there could easily be quite custom requirements about which bundles get produced. Would a plugin approach be appropriate here? Ie. the base case of producing a esm bundle is supplied out of the box and any additional bundles can be supported using a plugin?

Reply:
We could also add another attribute where the value hints if the asset belongs to the content or fallback.

Reply:
That sounds like a good generic approach from the POV of Podium methinks. Will whip something up